src/domain.types/statistics/daily.statistics/daily.statistics.dto.ts

```diff
@@ -0,0 +1,16 @@
+export interface YearWiseUsers{
+    Year?:number;
+    UserCount?:number;
+}
+
+export interface YearWiseDeviceDetails{
+    Year?:number;
+    DeviceDetails?:any[];
+}
+
+export interface DailyStatisticsDto {
+    id?                              : string;
+    StatisticsReportedDate?          : Date;
+    CronSchedulerTime?               :Date;
+    StatisticsData?                  : object;
```

Comment:
Change ' StatisticsData' to just Statistics everywhere.

Why this is an object?
If you do not know or want to Typify it, make it **any** or **unknown**.